lib/node_modules/@stdlib/blas/ext/base/sasumpw/manifest.json

```diff
@@ -34,7 +34,12 @@
 				"-lm"
 			],
 			"libpath": [],
-			"dependencies": []
+			"dependencies": [
```

Comment:
This configuration file needs to be updated to include additional build configurations. See `@stdlib/blas/ext/base/dfill` for an example.